e2e-setup/astro-project/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Astro Runtime includes the following pre-installed providers packages: https://www.astronomer.io/docs/astro/runtime-image-architecture#provider-packages
+
+astro-run-dag # This package is needed for the astro run command. It will be removed before a deploy
```

Comment:
Do we need this if we're not yet testing that command? What does it mean by removing this before a deploy?

Reply:
This is just a default Astro project created by Astro CLI, without any changes, so I don't see a harm in keeping it. I believe CLI has some logic for updating requirements.txt on the fly during `deploy` (not a big fan), and this might be a result of that.